src/screens/SwitchMerchant/SwitchMerchant.res

```diff
@@ -285,7 +284,7 @@ let make = (~userRole, ~isAddMerchantEnabled=false) => {
       let token = HyperSwitchAuthUtils.parseResponseJson(
         ~json=res,
         ~email=responseDict->LogicUtils.getString("email", ""),
-        ~isAcceptInvite=acceptInvite,
+        ~isAcceptInvite=true,
```

Comment:
same here

Reply:
```
 ~isAcceptInvite=true,
```
Please check locally via removing it. I guess we can remove this then if every thing is working previously it was feature flag associated that's why I kept like that.

Reply:
Updated